docs/installation/environment-variables.md

```diff
@@ -226,3 +226,15 @@ For detailed information see [WaveMoney Developer Documentation](https://partner
 | Environment Variable | Default                              | Description                                                                                                               |
 | -------------------- | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------- |
 | `BINGMAP_API_URL`    | **Required** (when BingMaps is used) | BingMaps API URL (including the API key). For example `https://dev.virtualearth.net/REST/v1/Imagery/Metadata/Aerial?key=` |
+
+#### AfricasTalking
+
+For detailed information see [AfricasTalking Developer Documentation](https://developers.africastalking.com/tutorials?products=sms)
+
+| Environment Variable    | Default                            | Description                                                                     |
+| ----------------------- | ---------------------------------- | ------------------------------------------------------------------------------- |
+| `API_KEY`               | **Required** (when plugin is used) | AfricasTalking API key used for communication with AfricasTalking API.          |
+| `USER_NAME`             | **Required** (when plugin is used) | AfricasTalking username used for communication with AfricasTalking API.         |
+| `SHORT_CODE`            | **Required** (when plugin is used) | The shortcode you specified will appear as the sender on the recipient's phone. |
```

Comment:
Later in the code it looks like these are read from DB not from ENV. Could you explain?

Reply:
Is it possible to use both ways? If so, which takes priority?

Reply:
its not possible to use both ways.